tests/test_experimental_newlines.py

```diff
@@ -7,13 +7,7 @@
     __file__), '../examples/yarn2/experimental-newlines.csv'), 'r')
 
 runner_normal = YarnRunner(compiled_yarn_f, names_csv_f)
-
-# reset file position
-compiled_yarn_f.seek(0, 0)
-names_csv_f.seek(0, 0)
-
-runner_experimental = YarnRunner(
-    compiled_yarn_f, names_csv_f, experimental_newlines=True)
```

Comment:
Was seeking to reset this file no longer necessary?

Reply:
Ah, I didn't catch that you had moved this inside the runner. I think that's fine.